forge-gui/res/cardsfolder/upcoming/argivian_welcome.txt

```diff
@@ -0,0 +1,8 @@
+Name:Argivian Welcome
+ManaCost:W U
+Types:Instant
+A:SP$ Destroy | ValidTgts$ Creature.powerGE4 | TargetMin$ 0 | TargetMax$ 1 | TgtPrompt$ Select up to one target creature with power 4 or greater | SubAbility$ DBChoose | SpellDescription$ Destroy up to one target creature with power 4 or greater. Choose a nonland card in your hand. It perpetually gains flash. 
```

Comment:
how does the stack description look when you cast this?

Reply:
![image](https://user-images.githubusercontent.com/67333662/205337936-877d76a4-b248-4020-8f1e-5cdea1c937a6.png)

Reply:
I guess Stackdescription$ spelldescription would be better ?